plugins/database/redshift/redshift_test.go

```diff
@@ -189,71 +206,103 @@ func TestPostgreSQL_RenewUser(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 
-	statements := dbplugin.Statements{
-		Creation: []string{testRedshiftRole},
-	}
-
-	usernameConfig := dbplugin.UsernameConfig{
+	usernameConfig := dbplugin.UsernameMetadata{
 		DisplayName: "test",
 		RoleName:    "test",
 	}
 
-	username, password, err := db.CreateUser(context.Background(), statements, usernameConfig, time.Now().Add(2*time.Second))
+	const password = "SuperSecurePa55w0rd!"
+	newReq := dbplugin.NewUserRequest{
+		UsernameConfig: usernameConfig,
+		Password:       password,
+		Statements:     dbplugin.Statements{Commands: []string{testRedshiftRole}},
+		Expiration:     time.Now().Add(5 * time.Second),
```

Comment:
I recommend pulling the expiration duration out to a variable so it can be referenced below.